### PR TITLE
Update promotions.md

### PR DESCRIPTION
**Pull request by rodrini-the-po**
Added a link to attribute configuration to better inform users copied from price rule conditions user guide

## Purpose of this pull request

This pull request (PR) adds a link to another part of the documentation to inform customers on how to configure the conditions rule builder using product attributes

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/marketing/promotions.html
- ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/marketing/promotions.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ A rule is a collection of conditions (one or more) that apply changes in prices
 
 Conditions are statements that refine the list of products and situations for applying the rule. The attributes and options for conditions differ between the types of available rules. When met, the action is completed such as discounts, buy one get one free (BOGO), and other options. Rules can be as simple or as complicated as needed to match your business needs, seasonal discounts and promotions, and year long opportunities. For example, you may want to add a few more options for the holidays while providing free shipping year-round when carts have a high subtotal.
 
+{:.bs-callout .bs-callout-info}
+If you want to define a condition based on a specific product attribute, **Use for Promo Rule Conditions** for the attribute must be set to `Yes` in your [Storefront Properties]({% link stores/attribute-product-create.md %}) for the attribute.
```
Comment (leslietilling) [inline src/marketing/promotions.md:16]: slight edit

```suggestion
If you want to define a condition based on a specific product attribute, **Use for Promo Rule Conditions** must be set to `Yes` for the attribute in your [Storefront Properties]({% link stores/attribute-product-create.md %}).
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just a slight edit to make the sentence read a little clearer.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**